test/integration/custom-routes/next.config.js

```diff
@@ -226,6 +239,19 @@ module.exports = {
             },
           ],
         },
+        {
+          source: '/named-pattern/:path(.*)',
+          headers: [
+            {
+              key: 'x-something',
+              value: 'value=:path',
+            },
+            {
+              key: 'path-:path',
```

Comment:
What's the practical case for setting the header name to a dynamic value?

Reply:
Depending on their set-up a user might want to return the original path as a header for rewrite/proxying scenarios. We mainly added this in `@now/routing-utils` for consistency so added this here to match that behavior